__tests__/app.test.js

```diff
@@ -99,3 +98,51 @@ describe("GET /api/topics", () => {
         });
     });
   });
+
+  describe('GET /api/articles/:article_id/comments', () => {
+    test('200: returns array of comments for given article_id', () => {
+      return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body
+        expect(comments.length).toBe(11)
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number)
+          })
+        })
+        })
+      })
+      test('200: returns array of comments ordered by most recent', () => {
+        return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body
+        expect(comments.length).toBe(11)
```

Comment:
Since this expect is the same in the above test there is no need to have it here again.